jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerContextGenerator.java

```diff
@@ -47,6 +46,18 @@
  */
 public class DockerContextGenerator {
 
+  /** The template to generate the Dockerfile from. */
+  private static final String DOCKERFILE_TEMPLATE =
```

Comment:
If we're changing to just building the string in the code, we can just use stringbuilder and append the parts rather than doing the template replacement.